mk/riscof.mk

```diff
@@ -0,0 +1,95 @@
+###############################################################################
+#
+# Copyright 2023 OpenHW Group
+# Copyright 2023 Dolphin Design
```

Comment:
This is a separate file.  If you are modifying the file substantially, please update the Copyright by adding "Copyright 2024 Silicon Labs".

Reply:
Added copyright to the riscof.mk file, this is based on the e40p work and modified slightly.